README.md

```diff
@@ -2,152 +2,316 @@ Plop
-[![plop on slack](https://plopjs-slack-ofhdlhmvql.now.sh/badge.svg)](https://plopjs-slack-ofhdlhmvql.now.sh/)
+&nbsp;
+[![plop on slack](https://plopjs-slack-rjjmojauhb.now.sh/badge.svg)](https://plopjs-slack-rjjmojauhb.now.sh/)
 
-![plop demo](http://i.imgur.com/penUFkr.gif)
+## What is Plop?
+Plop is a what I like to call a "micro-generator framework." Now, I call it that because it is a small tool that gives you a simple way to generate code or any other type of flat text files in a consistent way. You see, we all create structures and patterns in our code (routes, controllers, components, helpers, etc). These patterns change and improve over time so when you need to create a NEW *insert-name-of-pattern-here*, it's not always easy to locate the files in your codebase that represent the current "best practice." That's where plop saves you. With plop, you have your "best practice" method of creating any given pattern in CODE. Code that you can be easily run from the terminal by simply typing `plop`. Not only does this save you from hunting around in your codebase for the right files to copy, but it also turns "the right way" into "the easiest way" to make new files.
 
-Plop is essentially glue code between [inquirer](https://github.com/SBoudrias/Inquirer.js/) prompts and [handlebar](https://github.com/wycats/handlebars.js/) templates. You can also add your own handlebar helper methods and use them in your templates.
+If you boil plop down to its core, it is basically glue code between  [inquirer](https://github.com/SBoudrias/Inquirer.js/) prompts and [handlebar](https://github.com/wycats/handlebars.js/) templates.
 
-# Getting Started
-## 1. Install plop globally
+> This documentation is a work in progress. If you have great ideas, I'd love to hear them.
+
+## Installation
+### 1. Add plop to your project
 ```
-$ npm install -g plop
+$ npm install --save-dev plop
 ```
-## 2. Install plop in your project's devDependencies
+### 2. Install plop globally (optional, but recommended for easy access)
 ```
-$ npm install --save-dev plop
+$ npm install -g plop
 ```
-## 3. Create a plopfile.js at the root of your project
+### 3. Create a plopfile.js at the root of your project
 ``` javascript
 module.exports = function (plop) {
 	// create your generators here
 	plop.setGenerator('basics', {
 		description: 'this is a skeleton plopfile',
-		prompts: [],
-		actions: []
+		prompts: [], // array of inquirer prompts
+		actions: []  // array of actions
 	});
 };
 ```
----
-# Anatomy of a Plop Generator
-The main parts of a plop generator are the plop file (`plopfile.js`) and the templates. Templates can either be inline, or in separate files.
 
-A basic plop file starts its life as a lowly node module that exports a function that accepts the `plop` object.
+## Your First Plopfile
+A plopfile starts its life as a lowly node module that exports a function that accepts the `plop` object as its first parameter.
```

Comment:
Suggestion:
* _a function which accepts_

To prevent repetition of "_that does this that does that_" which sounded complex to me at first glance.